src/sections/Community/Member-single/index.js

```diff
@@ -131,6 +140,12 @@ const MemberSingle = ({frontmatter}) => {
               </div>
             </Col>
           </Row>
+          <Row Hcenter>
+            <Col className="start-contributing" sm={12} lg={12}>
+              <p> Would you Like a Member Profile?</p>
+              <Button primary title="Start Contributing Today" url="/community"/>
```

Comment:
@leecalcote what should be this link?

Reply:
Ultimately, I think this is one of those areas where what we want to do is point people to a form to enter their email address into you, so that we can send them an invite to slack and to all of the rest of the resources in the community.  @Nikhil-Ladha, if you’re comfortable making another mailing list audience in Mailchimp...

Reply:
Okay. This is be merged for now then